.github/workflows/integration-sqlite.yml

```
@@ -0,0 +1,169 @@
+name: Integration sqlite
+
+on:
+  pull_request:
+    paths:
+      # Only running on PR for this file to save CI time (otherwise pgsql only)
+      - '.github/workflows/integration-sqlite.yml'
+
+  push:
+    branches:
+      - main
+      - master
+      - stable*
+
+permissions:
+  contents: read
+
+concurrency:
+  group: integration-sqlite-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+
+    outputs:
+      # FIXME src: ${{ steps.changes.outputs.src}}
+      src: 'true'
+
+    steps:
+      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
+        id: changes
+        continue-on-error: true
+        with:
+          filters: |
+            src:
+              - '.github/workflows/**'
+              - 'appinfo/**'
+              - 'lib/**'
+              - 'templates/**'
+              - 'tests/integration/**'
+              - 'vendor/**'
+              - 'vendor-bin/**'
+              - '.php-cs-fixer.dist.php'
+              - 'composer.json'
+              - 'composer.lock'
```

Should be adjusted to match also apps/ subdirectories
```suggestion
             - '.github/workflows/**'
             - '3rdparty/**'
             - '**/*.php'
             - '**/lib/**'
             - '**/tests/**'
             - '**/vendor-bin/**'
             - '.php-cs-fixer.dist.php'
             - 'composer.json'
            - 'composer.lock'
```

---

yeah copy pastes from talk for now.

also needs `build/integration` added

---

Out of curiosity but why are the integration tests located in `build/` rather then `tests/`?

---

Because the coworker doing it back at owncloud times put it there.
I would be fine with moving it, but it "breaks" local setups/scripts and open PRs (but there shouldn't be too many), so maybe we use this "refactoring" as the right point in time to do it?